src/odemis/odemisd/modelgen.py

```diff
@@ -808,7 +839,7 @@ def instantiate_component(self, name):
         # Add to the microscope all the new components that should be child
```

Comment:
You can update the docstring (l. 818 -> 820), to replace "children that are created by separate instantiation" by "dependencies". And drop the "if they are created by delegation".